imnodes.cpp

```diff
@@ -1772,6 +1778,15 @@ void EndNode()
     g.canvas_draw_list->ChannelsMerge();
 }
 
+ImVec2 GetNodeDimensions(int node_id)
+{
+    EditorContext& editor = editor_context_get();
+	const int node_idx = editor.nodes.find(node_id);
+    assert(node_id != -1); //invalid node_id
```

Comment:
I think it would make more sense for the assert to be inside `find()`, since that is where we pass the `-1` magic number to GetInt

Reply:
this assert is actually invalid (as-in not what I intended).
is should have been `assert(node_idx != -1)`  (node_idx, not node_id).
While this check roughly the same ("is a valid node_id is passed as argument"), 7a217e5 fixed this

Reply:
Oh yeah good catch, totally missed that 😄